index.html

```diff
@@ -3,8 +3,26 @@
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
+    <link rel="stylesheet" href="styles/styles.css">
 </head>
 <body>
-
+<h1>What do we offer</h1>
+<div>
+    <div class="support">
+        <img src="img/heads.jpg" alt="heads">
+        <p>Support 24/7</p></div>
```

Comment:
The closing tag should be on the same level as the opening tag. It's a good style of code.